util/Logger.js

```diff
@@ -8,43 +8,31 @@ class Logger {
   static log(content, type = "log") {
     const timestamp = `[${moment().format("YYYY-MM-DD HH:mm:ss")}]:`;
     switch (type) {
-      case "log": {
-        return console.log(`${timestamp} ${chalk.bgBlue(type.toUpperCase())} ${content} `);
-      }
-      case "warn": {
-        return console.log(`${timestamp} ${chalk.black.bgYellow(type.toUpperCase())} ${content} `);
-      }
-      case "error": {
-        return console.log(`${timestamp} ${chalk.bgRed(type.toUpperCase())} ${content} `);
-      }
-      case "debug": {
-        return console.log(`${timestamp} ${chalk.green(type.toUpperCase())} ${content} `);
-      }
-      case "cmd": {
-        return console.log(`${timestamp} ${chalk.black.bgWhite(type.toUpperCase())} ${content}`);
-      }
-      case "ready": {
-        return console.log(`${timestamp} ${chalk.black.bgGreen(type.toUpperCase())} ${content}`);
-      } 
+      case "log": return console.log(`${timestamp} ${chalk.bgBlue(type.toUpperCase())} ${content} `);
```

Comment:
this is kinda redundant you maybe could solve this another way

Reply:
@Dev-Yukine What do you mean? I simply deduced unneeded LOC's

Reply:
rewrite the whole logic of this, all these cases are 95% simular and have the format "timestamp type_with_color content"

Reply:
well I was going to rewrite it anyway so yeah